Gemfile

```diff
@@ -113,6 +113,8 @@ gem 'ofn-qz', github: 'openfoodfoundation/ofn-qz', branch: 'ofn-rails-4'
 
 gem 'good_migrations'
 
+gem "view_component", require: "view_component/engine"
```

Comment:
Since ViewComponent will be used to render the component in both the development and production environment, I guess not. But maybe I'm wrong about this group concept. 
See exemple from Github PrimerViewComponents: https://github.com/primer/view_components/blob/740cd4821e544bbcdca5f1d65f07468cde8457a1/Gemfile#L36

Reply:
Yep, it should stay there :+1: It'll be used in all environments.

Reply:
Ah, yes, that sounds right! Storybook should go into development.